cluster/gce/config-default.sh

```diff
@@ -96,22 +96,10 @@ export NODE_SERVICE_ACCOUNT=${KUBE_GCE_NODE_SERVICE_ACCOUNT:-default}
 # KUBELET_TEST_ARGS are extra arguments passed to kubelet.
 export KUBELET_TEST_ARGS=${KUBE_KUBELET_EXTRA_ARGS:-}
 
-# Default container runtime
-export CONTAINER_RUNTIME=${KUBE_CONTAINER_RUNTIME:-containerd}
-# Default container runtime for windows
-export WINDOWS_CONTAINER_RUNTIME=${KUBE_WINDOWS_CONTAINER_RUNTIME:-containerd}
-
-# Set default values with override
-if [[ "${CONTAINER_RUNTIME}" == "docker" ]]; then
-  export CONTAINER_RUNTIME_ENDPOINT=${KUBE_CONTAINER_RUNTIME_ENDPOINT:-unix:///var/run/dockershim.sock}
-  export CONTAINER_RUNTIME_NAME=${KUBE_CONTAINER_RUNTIME_NAME:-docker}
-  export LOAD_IMAGE_COMMAND=${KUBE_LOAD_IMAGE_COMMAND:-}
-elif [[ "${CONTAINER_RUNTIME}" == "containerd" ||  "${CONTAINER_RUNTIME}" == "remote" ]]; then
```

Comment:
we might not want to remove the if condition with containerd, as this script might be used with some other container runtimes, not fully sure on this but there is no harm in keeping it as well
```
if [[ "${CONTAINER_RUNTIME}" == "containerd"
```

this is really weird is `CONTAINER_RUNTIME` here not same as --container-runtime kubelet flag? because kubelet flag cannot be set to `containerd`

Reply:
The new PR for scripts is https://github.com/kubernetes/kubernetes/pull/107663.

The `"${CONTAINER_RUNTIME}" == "containerd"` was mistake done at some point. This is why this hack was needed:
https://github.com/kubernetes/kubernetes/blob/94ff5ea1f1727826c9848878c890aa793f733aec/test/e2e/framework/test_context.go#L489-L493

If anyone wants to use a different CRI, should be enough to set `KUBE_CONTAINER_RUNTIME_NAME`, `KUBE_CONTAINER_RUNTIME_ENDPOINT` and  `KUBE_LOAD_IMAGE_COMMAND`.

Reply:
Thanks for the explantation @hakman sounds good!!